migration-engine/connectors/sql-migration-connector/src/sql_schema_calculator/sql_schema_calculator_flavour/postgres.rs

```diff
@@ -15,6 +15,19 @@ impl SqlSchemaCalculatorFlavour for PostgresFlavour {
             .collect()
     }
 
+    fn column_type_for_unsupported_type(&self, field: ScalarFieldWalker<'_>, description: String) -> sql::ColumnType {
+        sql::ColumnType {
+            full_data_type: description.clone(),
+            family: sql::ColumnTypeFamily::Unsupported(description),
+            arity: match field.ast_field().arity {
+                ast::FieldArity::Required => sql::ColumnArity::Required,
+                ast::FieldArity::Optional => sql::ColumnArity::Nullable,
+                ast::FieldArity::List => sql::ColumnArity::List,
+            },
+            native_type: None,
+        }
+    }
```

Comment:
I don't understand this change — unsupported types are not involved in the test, why is this necessary?